packages/autocannon-ui-frontend/src/components/ProgressBar.jsx

```diff
@@ -23,9 +26,11 @@ export default function ProgressBar(props) {
     <div className={classes.root}>
       <Box display="flex" alignItems="center" flexDirection="column">
         <Box minWidth={35}>
-          <Typography variant="body2" color="textSecondary">{`${Math.round(
-            props.value
-          )}%`}</Typography>
+          <Typography
+            variant="body2"
+            color="textSecondary"
+            className={classes.percentLabel}
```

Comment:
not a big deal to use a CSS class, but since there is a property for that we can stay more consistent and just use properties for styling, unless what we need to do isn't possible with properties. see https://material-ui.com/api/typography/